configs/_base_/models/hv_pointpillars_secfpn_kitti.py

```diff
@@ -34,12 +35,13 @@
         in_channels=384,
         feat_channels=384,
         use_direction_classifier=True,
+        assign_per_class=True,
         anchor_generator=dict(
             type='Anchor3DRangeGenerator',
             ranges=[
-                [0, -39.68, -0.6, 70.4, 39.68, -0.6],
-                [0, -39.68, -0.6, 70.4, 39.68, -0.6],
-                [0, -39.68, -1.78, 70.4, 39.68, -1.78],
+                [0.08, -39.60, -0.6, 68.88, 39.44, -0.6],
```

Comment:
Try with AlignedAnchor3DRangeGenerator to make the range consistent with other settings